docs/compatibility.include

```diff
@@ -4,6 +4,7 @@ django CMS   Python                                  Django
 \            3.12 3.11 3.10 3.9 3.8 3.7 3.6  5.0 4.2 4.1 4.0 3.2 3.1 3.0 2.2
 ===========  ==== ==== ==== === === === ===  === === === === === === === ===
 4.1.x        ✓    ✓    ✓    ✓   ×   ×   ×    ✓   LTS ✓   ✓   LTS ×   ×   ×
+3.11.5+      ✓    ✓    ✓    ✓   ✓   ✓   ×    ×   LTS ✓   ✓   LTS ×   ×   ×
```

Comment:
```suggestion
3.11.6+      ✓    ✓    ✓    ✓   ✓   ✓   ×    ✓   LTS ✓   ✓   LTS ×   ×   ×
```
Since 3.11.6, also django 5.0 is supported. Shall we also add this?